scripts/test-goose-streaming.sh

```
@@ -0,0 +1,80 @@
+  echo "Raw output:"
+  cat "$OUTFILE"
+  exit 1
+fi
+echo "Session ID: $SESSION_ID"
+
+# 3. Send a short prompt
+echo "{\"jsonrpc\":\"2.0\",\"id\":3,\"method\":\"session/prompt\",\"params\":{\"sessionId\":\"$SESSION_ID\",\"prompt\":[{\"type\":\"text\",\"text\":\"Say hello in exactly one sentence.\"}]}}" >&3
+
+# Wait for response (up to 30s)
+echo "Waiting for response..."
+for i in $(seq 1 60); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)